lib/modules/address.js

```diff
@@ -172,7 +172,10 @@ AddressModule.prototype.subscribe = function(name, emitter, addresses) {
 
 AddressModule.prototype.unsubscribe = function(name, emitter, addresses) {
   $.checkArgument(emitter instanceof EventEmitter, 'First argument is expected to be an EventEmitter');
-  $.checkArgument(Array.isArray(addresses), 'Second argument is expected to be an Array of addresses');
```

Comment:
We could check these cases: `Array.isArray(addresses) || _.isUndefined(addresses)`